### PR TITLE
Support new format doctag

### PR DESCRIPTION
**Pull request by OlegKipchatov**

### CODE REVIEW

File: templates/component.tst [hunk 1]
```diff
@@ -69,9 +69,14 @@ const CUSTOM_VALUE_ACCESSOR_PROVIDER = {
...
-<#? it.isDeprecated #>/** @deprecated */<#?#>
-<#? it.docID #>/** @name <#= it.docID #> */<#?#>
+<#? it.isDeprecated #>
+/**
+ * <-<#= it.docID #>->
+ * @deprecated <-<#= it.docID #>:depNote->
+ */
+<#?? it.docID #>
+/** <-<#= it.docID #>-> */
+<#?#>
```
Comment (IlyaKhD) [inline templates/component.tst:79]: What is the result for `it.isdeprecated === true` and `it.docID === undefined` ?
I guess
```
/**
 * <-->
 * @deprecated <-:depNote->
 */
```

I suggest smth like:
```suggestion
<#? it.docID #>
/**
 * <-<#= it.docID #>-><#? it.isDeprecated #>
 * @deprecated <-<#= it.docID #>:depNote-><#?#>
 */
<#?#>
```

Comment (OlegKipchatov) [reply templates/component.tst:79]: DocID exist always, I would change to this:
```
<#? prop.isDeprecated #>
/**
 * <-<#= prop.docID #>->
 * @deprecated <-<#= prop.docID #>:depNote->
 */
<#??#>
/** <-<#= prop.docID #>-> */
<#?#>
```

Comment (IlyaKhD) [reply templates/component.tst:79]: Ok

What about having multiline comment in both cases?

```
/**
 * <-<#= prop.docID #>->
 */
```

Comment (OlegKipchatov) [reply templates/component.tst:79]: I see no reason not to do this, but I was thinking about minimizing the output size

Comment (IlyaKhD) [reply templates/component.tst:79]: The reason is to have no differences.

What size do you want to minimize? The npm packages size?
Comments are not included into the run-time bundle.

**Pull request merged**